src/engine/sys/sdl_glimp.cpp

```diff
@@ -2033,9 +2033,21 @@ static void GLimp_InitExtensions()
 
 	logger.Notice("...using shading language version %i", glConfig2.shadingLanguageVersion );
 
-	// Texture formats and compression
+
+	// OpenGL driver constants.
+
+	glGetIntegerv( GL_MAX_COMBINED_TEXTURE_IMAGE_UNITS, &glConfig2.maxTextureUnits );
+	glGetIntegerv( GL_MAX_TEXTURE_SIZE, &glConfig.maxTextureSize );
+	glGetIntegerv( GL_MAX_3D_TEXTURE_SIZE, &glConfig2.max3DTextureSize );
 	glGetIntegerv( GL_MAX_CUBE_MAP_TEXTURE_SIZE_ARB, &glConfig2.maxCubeMapTextureSize );
 
+	logger.Notice( "...using up to %d texture size.", glConfig.maxTextureSize );
+	logger.Notice( "...using up to %d 3D texture size.", glConfig2.max3DTextureSize );
+	logger.Notice( "...using up to %d CubeMap texture size.", glConfig2.maxCubeMapTextureSize );
```

Comment:
Weird capitalization here. "Cubemap" is usually treated as 1 word

Reply:
Well, we have both:

```
$ rg '[Cc]ubeMap' | wc -l
87
$ rg '[Cc]ubemap' | wc -l
110
```

We can unify that later.

Reply:
I mean the capitalization should never be like that if you are writing text, not a function name or whatever. If you want it to be two words then "cube map".

Reply:
I fixed the capitalization.